Userland/Libraries/LibGfx/ICC/TagTypes.h

```diff
@@ -58,6 +59,46 @@ class UnknownTagData : public TagData {
     }
 };
 
+// ICC v4, 10.2 chromaticityType
+class ChromaticityTagData : public TagData {
+public:
+    static constexpr TagTypeSignature Type { 0x6368726D }; // 'chrm'
+
+    static ErrorOr<NonnullRefPtr<ChromaticityTagData>> from_bytes(ReadonlyBytes, u32 offset, u32 size);
+
+    // ICC v4, Table 31 — Colorant and phosphor encoding
+    enum class PhosphorOrColorantType : u16 {
+        Unknown = 0,
+        ITU_R_BT_709_2 = 1,
+        SMPTE_RP145 = 2,
+        EBU_Tech_3213_E = 3,
+        P22 = 4,
+        P3 = 5,
+        ITU_R_BT_2020 = 6,
+    };
+
+    static StringView phosphor_or_colorant_type_name(PhosphorOrColorantType);
+
+    struct xyCoordinate {
```

Comment:
The lower-case `xy` here is weird, but `xy` and `XY` coordinates are different things. So this is intentional.